Source/santad/DataLayer/WatchItemPolicy.h

```diff
@@ -120,7 +118,56 @@ struct WatchItemPolicy {
   std::optional<std::string> custom_message;
   std::optional<NSString *> event_detail_url;
   std::optional<NSString *> event_detail_text;
-  std::vector<Process> processes;
+};
+
+struct DataWatchItemPolicy : public WatchItemPolicyBase {
+  DataWatchItemPolicy(std::string_view n, std::string_view v, std::string_view p,
+                      WatchItemPathType pt = kWatchItemPolicyDefaultPathType,
+                      bool ara = kWatchItemPolicyDefaultAllowReadAccess,
+                      bool ao = kWatchItemPolicyDefaultAuditOnly,
+                      WatchItemRuleType rt = kWatchItemPolicyDefaultRuleType,
+                      bool esm = kWatchItemPolicyDefaultEnableSilentMode,
+                      bool estm = kWatchItemPolicyDefaultEnableSilentTTYMode,
+                      std::string_view cm = "", NSString *edu = nil, NSString *edt = nil,
+                      std::vector<WatchItemProcess> procs = {})
+      : WatchItemPolicyBase(n, v, ara, ao, rt, esm, estm, cm, edu, edt),
+        path(p),
+        path_type(pt),
+        processes(std::move(procs)) {}
+
+  bool operator==(const WatchItemPolicyBase &other) const override {
+    const DataWatchItemPolicy *otherPolicy = dynamic_cast<const DataWatchItemPolicy *>(&other);
+    if (!otherPolicy) {
+      return false;
+    }
+
+    // Now compare base and derived class attributes
+    return WatchItemPolicyBase::operator==(*otherPolicy) && path_type == otherPolicy->path_type &&
+           path == otherPolicy->path && processes == otherPolicy->processes;
+  }
+
+  bool operator!=(const WatchItemPolicyBase &other) const override { return !(*this == other); }
+
+  std::string path;
+  WatchItemPathType path_type;
+  std::vector<WatchItemProcess> processes;
+};
+
+struct ProcessWatchItemPolicy : public WatchItemPolicyBase {
```

Comment:
Note: This is very incomplete right now. Just meant to be a stub to start getting interfaces worked out.